docs/reference/commandline/run.md

```diff
@@ -65,6 +65,7 @@ parent = "smn_cli"
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
       --privileged=false            Give extended privileges to this container
+      --read-bps-device=""       Specifies upper limit on READ rate from the device
```

Comment:
looks like "Specifies" doesn't align with the other lines.

Perhaps we should use `Limit read rate from the device (in bits per second)` (or is it bytes per second?)

I wonder if we should be more specific about "device", it isn't clear to me *which* device; does it apply to any disk read (and write for `write-bps-device`)? Does it apply to volumes?